modules/rubiconBidAdapter.js

```diff
@@ -600,21 +624,31 @@ function _getScreenResolution() {
   return [window.screen.width, window.screen.height].join('x');
 }
 
-function _getDigiTrustQueryParams() {
+function _getDigiTrustQueryParams(bidRequest = {}, endpointName) {
+  if (!endpointName || !DIGITRUST_PROP_NAMES[endpointName]) {
+    return null;
+  }
+  const propNames = DIGITRUST_PROP_NAMES[endpointName];
+
   function getDigiTrustId() {
-    let digiTrustUser = window.DigiTrust && (config.getConfig('digiTrustId') || window.DigiTrust.getUser({member: 'T9QSFKPDN9'}));
+    const bidRequestDigitrust = utils.deepAccess(bidRequest, 'userId.digitrustid.data');
+    if (bidRequestDigitrust) {
+      return bidRequestDigitrust;
```

Comment:
check if privacy.optout==true and skip if so

Reply:
This check is being done in existing code, so I didn't change it.  `getDigiTrustId` is within the `_getDigiTrustQueryParams` function, which does that check before returning data.

Reply:
Here is the line: https://github.com/rubicon-project/Prebid.js/blob/1076efb83eb1eef7248d8f3b2ee65643355e4746/modules/rubiconBidAdapter.js#L645